runtime/src/snapshot_utils.rs

```diff
@@ -2049,6 +2587,22 @@ pub fn verify_snapshot_archive<P, Q, R>(
         assert!(crate::serde_snapshot::compare_two_serialized_banks(&p1, &p2).unwrap());
         std::fs::remove_file(p1).unwrap();
         std::fs::remove_file(p2).unwrap();
+
+        // The new the status_cache file is inside the slot directory together with the snapshot file.
```

Comment:
If we move the file, won't that break it if we run this verify multiple times? Since it won't bethere the second time.